docs/contivctl.md

```diff
@@ -0,0 +1,226 @@
+## contivctl
+
+contivctl command line tool lets you add/modify/delete objects into contiv object model using REST api.
+
+Command can be found under $GOPATH/bin directory (/opt/gopath/bin in vagrant VMs) after a build. 
+Currently, contivctl needs to be run on the same host where netmaster is running
+
+## Usage
+
+```
+vagrant@netplugin-node1:~$ contivctl --help
+usage: contivctl [-h] [--version] {policy,rule,group,network} ...
+
+positional arguments:
+  {policy,rule,group,network}
+    policy              Policy operations
+    rule                Rule add/delete
+    group               Endpoint group operations
+    network             Network operations
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --version             show program's version number and exit
+```
+
+### Creating networks
+
+```
+vagrant@netplugin-node1:~$ contivctl network create --help
+usage: contivctl network create [-h] [-tenantName TENANTNAME]
+                                [-public {yes,no}] [-encap {vlan,vxlan}]
+                                -subnet SUBNET -defaultGw DEFAULTGW
+                                networkName
+
+positional arguments:
+  networkName           Network name
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -tenantName TENANTNAME
+  -public {yes,no}      Is this a public network
+  -encap {vlan,vxlan}   Packet tag
+  -subnet SUBNET        Subnet addr/mask
+  -defaultGw DEFAULTGW  default GW
+```
+
+#### Examples
+
+Following creates a vxlan network called production under default tenant and assigns a subnet to it
```

Comment:
It will be nice to use back-quote for keywords/proper-nouns like `production` and `default`, same goes for examples below.